package.json

```diff
@@ -31,7 +31,7 @@
   "devDependencies": {
     "docsify-cli": "^4.4.2",
     "husky": "^4.3.8",
-    "jest": "^26.6.3",
+    "jest": "26.6.0",
```

Comment:
can we achieve the same without removing the semver specifier from the version number? my concern is that pinning the version will prevent future upgrades, which is what we want to prevent right now but we also want to be able to automatically do them at some point.

Reply:
So it turns that yes, we can have it there.

For whatever reason I was getting the same error when I left it there for the first time. It works now on a fresh installation. PR updated.